### PR TITLE
Remove SQLite as default option

### PR DESCRIPTION
**Pull request by Kukks**
IF MERGED, STORES WITH NO DB CONFIG WILL NOT START UNTIL CONFIG IS UPDATED WITH `sqlitefile=sqlite.db`

### CODE REVIEW

File: BTCPayServer/Configuration/DefaultConfiguration.cs [hunk 1]
```diff
@@ -25,8 +25,10 @@ protected override CommandLineApplication CreateCommandLineApplicationCore()
...
             app.Option("--allow-admin-registration", $"For debug only, will show a checkbox when a new user register to add himself as admin. (default: false)", CommandOptionType.BoolValue);
             app.Option("--chains | -c", $"Chains to support as a comma separated (default: btc; available: {chains})", CommandOptionType.SingleValue);
-            app.Option("--postgres", $"Connection string to a PostgreSQL database (default: SQLite)", CommandOptionType.SingleValue);
-            app.Option("--mysql", $"Connection string to a MySQL database (default: SQLite)", CommandOptionType.SingleValue);
+            app.Option("--postgres", $"Connection string to a PostgreSQL database", CommandOptionType.SingleValue);
+            app.Option("--mysql", $"Connection string to a MySQL database", CommandOptionType.SingleValue);
+            app.Option("--sqlite", $"Connection string to a SQLite database", CommandOptionType.SingleValue);
+            app.Option("--sqlitefile", $"File name to an SQLite database file inside the data directory", CommandOptionType.SingleValue);
```
Comment (NicolasDorier) [inline BTCPayServer/Configuration/DefaultConfiguration.cs:31]: this is confusing to have two parameters. Best is to keep only `--sqlite` and put an example in the description.

Comment (Kukks) [reply BTCPayServer/Configuration/DefaultConfiguration.cs:31]: that one requires that you have a filepath which could break if you change your data dir and your db was in there.
I would either keep sqlitefile only or both

Comment (NicolasDorier) [reply BTCPayServer/Configuration/DefaultConfiguration.cs:31]: then let's keep sqlitefile

Comment (Kukks) [reply BTCPayServer/Configuration/DefaultConfiguration.cs:31]: done

**Pull request merged**